Text/CustomEntitySearch/README.md

```diff
@@ -0,0 +1,116 @@
+---
+topic: sample
+languages:
+- csharp
+products:
+- azure-cognitive-services
+name: Custom Entity Search sample skill for cognitive search
+description: This custom skill finds user defined entities in given texts.
+azureDeploy: https://raw.githubusercontent.com/Azure-Samples/azure-search-power-skills/master/Text/CustomEntitySearch/azuredeploy.json
+---
+
+# CustomEntitySearch
+
+This custom skill finds finds user defined entities in given texts.
+
+[![Deploy to Azure](https://azuredeploy.net/deploybutton.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure-Samples%2Fazure-search-power-skills%2Fmaster%2FText%2FCustomEntitySearch%2Fazuredeploy.json)
+
+## Requirements
+
+These skills have no additional requirements than the ones described in [the root `README.md` file](../../README.md).
+
+## Settings
+
+This function requires Latin-based text (as seen in the sample document provided).
+
+## Sample Input:
+
+```json
+{
+    "values": [
+        {
+            "recordId": "1",
+            "data":
+            {
+                "text":  "Learn how to leverage Azure Storage in your applications with our quickstarts and tutorials.",
+				"words": [
+					"learn",
+					"app"
+				]
+            }
+        },
+        {
+            "recordId": "foo1",
+            "data":
+            {
+                "text":  "Azure Storage includes Azure Blobs (objects), Azure Data Lake Storage Gen2, Azure Files, Azure Queues, and Azure Tables.",
+				"words": [
+					"bing"
+				]
+            }
+        }
+    ]
+}
+```
+
+## Sample Output:
+
+```json
+{
+    "values": [
+        {
+            "recordId": "1",
+            "data": {
+				"entities": [
+					{
+						"name": "learn",
+						"matchIndex": 1
+					},
+					{
+						"name": "app",
+						"matchIndex": 45
+					}
+			]
+        },
+        {
+            "recordId": "foo1",
+            "data": 
+			{
+				"entities": [
+					{
```

Comment:
Please make sure indentation style is consistent internally (you have a mix of tabs and spaces) and with other samples' READMEs.